modules/dream2nix/buildRustPackage/patch-workspace-deps.jq

```diff
@@ -0,0 +1,48 @@
+def normalizeWorkspaceDep:
```

Comment:
Where is this file used?

Reply:
in `vendor.nix`. this should share the same file in the `rust-crane` folder but i didn't know where to put shared files between modules, so i kept it like this